src/libs/cropOrRotateImage/index.native.js

```diff
@@ -0,0 +1,24 @@
+import RNImageManipulator from '@oguzhnatly/react-native-image-manipulator';
+import RNFetchBlob from 'rn-fetch-blob';
+
+/**
+ * Crops and rotates the image on ios/android
+ *
+ * @param {String} uri
+ * @param {Array<Object>} actions
+ * @param {Object} options
+ * @returns {Promise<Object>} Returns cropped and rotated image
+ */
+function cropOrRotateImage(uri, actions, options = {}) {
+    return new Promise((resolve) => {
+        RNImageManipulator.manipulate(uri, actions, options).then((result) => {
+            RNFetchBlob.fs.stat(result.uri.replace('file://', '')).then(({size}) => {
+                resolve({
+                    ...result, size, type: 'image/png', name: `${options.name || 'fileName'}.jpg`,
```

Comment:
... `type: 'image/png'` here?

Reply:
I think this is a mistake.

Reply:
Ya thought so - I don't believe it makes a difference at the moment - to solve https://github.com/Expensify/App/issues/11063 we'll most likely try to make this type dynamic